CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+## [2.9.6] - 2024-05-17
```

Comment:
I don't know about the empty unreleased?

![image](https://github.com/pluginsGLPI/escalade/assets/8530352/695237fa-6865-45df-8f02-a1f2282913bf)

Reply:
useless for the moment, I withdraw it (we'll add it to the next PR)